### ISSUE TITLE
Feature req: Don't delete just edit

### ISSUE THREAD

**Issue by jhchrist** [andrewbanchich/shreddit#61]
Would very much rather leave reddit with 10k comments of garbled nonsense than just having the comments be removed.

**Comment by andrewbanchich** [comment]
I agree, and plan on adding this once a fix or workaround is in place for #35 . Until then, if comments are only edited, any subsequent runs won't progress to the next set of comments and will just edit the same ones over and over.

**Comment by andrewbanchich** [comment]
Yes, if you have that export it should not run into this problem.

**Comment by SuspectTyrannosaurus** [comment]
In that case, as a 'hack', would it be as simple as commenting out [the "delete" call in mod.rs](https://github.com/andrewbanchich/shreddit/blob/1d0c045211fb7628de6db37acdda56a0b806ea4a/src/things/mod.rs#L44) ? To make the program edit the comments and not delete them?

(Then I'd just have to edit the Lorem Ipsum variable a couple lines down to make it leave something worth saying.)

**Comment by andrewbanchich** [comment]
Exactly!

**Comment by jhchrist** [comment]
Is is possible to even delete more than 1000 comments now without the gdpr export? still waiting for mine

**Comment by andrewbanchich** [comment]
Currently there is some issue with Reddit's list API not returning everything, even when paginating through results, so you would need to run shreddit multiple times until everything is deleted. It does work, it's just a bit of an annoyance that it doesn't work in one go.

**Comment by RFBomb** [comment]
I had already messed with the code to get this function, but #35 cropped up and we put it on hold. 
Since GDPR export seems to be a workaround, I submitted the pr for this, to include a note.

**Comment by andrewbanchich** [comment]
Added in [0.8.0](https://github.com/andrewbanchich/shreddit/releases/tag/v0.8.0). Thanks @RFBomb !
